src/tray_launcher/child_script.py

```diff
@@ -5,32 +5,47 @@
 import time as _t
 from pathlib import Path
 
+import psutil as _ps
+
 
 class ChildScript:
     ENCODING = "utf-8"
 
-    def __init__(self, script_path_str, logging_log):
-        self.script_path_str = script_path_str
-        self.script_path = Path(self.script_path_str)
-        self.child_script = None
-        self.child_script_PID = -1
-        self.current_PIDs = []
+    def __init__(self, pid, create_time, script_path, logging_log):
+        """Create a ChildScript instance, used in reattaching processes when the tray launcher restarts.
+
+        Args:
+            pid: int
+            create_time: float
+            script_path: Path
+            logging_log: Path
+        """
```

Comment:
Can you add more description on what each argument is. For example, "create_time: float, time of when script was initialized in [seconds?]"